servicecontrol/hosting.md

```diff
@@ -0,0 +1,47 @@
+---
+title: ServiceControl Hosting Options
+summary: Hosting options for running ServiceControl instances
+component: ServiceControl
+reviewed: 2025-03-20
+---
+
+ServiceControl instances can be hosted on:
+
+- Windows Server (virtual machines)
+- Linux containers
+
+
+Deployment options:
+
+- [Monitoring instances](/servicecontrol/monitoring-instances/deployment/)
+- [Audit instances](/servicecontrol/audit-instances/deployment/)
+- [Error instances](/servicecontrol/servicecontrol-instances/deployment/)
+
+## Windows Server
+
+Instances can be installed on Windows Server (virtual) machines using:
+
+- Commandline: ServiceControl Powershell Module
+- Management UI: ServiceControl Management Utility
```

Comment:
Can we add links here? The cumbersome thing is that we would have to add different links for each instance type.